lua/lspconfig/lemminx.lua

```diff
@@ -0,0 +1,51 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+local name = "lemminx"
+local bin_name = "org.eclipse.lemminx-uber.jar"
+
+configs[name] = {
+  default_config = {
+    cmd = {
+      "java", "-jar", bin_name
```

Comment:
I would just copy the bin_name in here

Reply:
done

Reply:
I'm confused by that amendment. For me, if I put the jar on my PATH, the result is this:

```
❱ org.eclipse.lemminx-uber.jar    
zsh: exec format error: org.eclipse.lemminx-uber.jar
```

Could it ever work? Maybe with a specifically prepared (hashbanged) jarfile?

Reply:
If you leave the `cmd` as `bin_name`, its easy enough to instead use the binary version of LemMinX that's available here: https://download.jboss.org/jbosstools/vscode/stable/lemminx-binary

Reply:
Ah, that's great. Maybe update the description field to mention that? Currently it gives instructions for downloading the non-executable jar file. Doesn't need to be full instructions IMO

Reply:
yeah, jar's are required to be run with the java-binary as far as I know. so `java -jar org.eclipse.lemminx-uber.jar` in that case.

I changed to the binary-method in stead as suggested. I didn't know that existed